In-house patch to remove unnecessary dependencies from Nova's
requirements files. The specific reasons are as follows:

babel			Not applicable

oslo.rootwrap		Not applicable to Solaris

paramiko		Not applicable

python-cinderclient	Upgrade to version 1.3.1

--- nova-2015.1.2/nova.egg-info/requires.txt.orig	2015-10-13 08:00:30.000000000 -0700
+++ nova-2015.1.2/nova.egg-info/requires.txt	2015-12-31 19:53:42.083286139 -0800
@@ -13,12 +13,10 @@ PasteDeploy>=1.5.0
 Paste
 sqlalchemy-migrate!=0.9.8,<0.10.0,>=0.9.5
 netaddr>=0.7.12
-paramiko>=1.13.0
 pyasn1
-Babel>=1.3
 iso8601>=0.1.9
 jsonschema<3.0.0,>=2.0.0
-python-cinderclient<1.2.0,>=1.1.0
+python-cinderclient<=1.3.1,>=1.1.0
 python-neutronclient<2.5.0,>=2.4.0
 python-glanceclient<0.18.0,>=0.15.0
 six>=1.9.0
@@ -31,7 +29,6 @@ oslo.log<1.1.0,>=1.0.0 # Apache-2.0
 oslo.serialization<1.5.0,>=1.4.0 # Apache-2.0
 oslo.utils!=1.4.1,<1.5.0,>=1.4.0 # Apache-2.0
 oslo.db<1.8.0,>=1.7.0 # Apache-2.0
-oslo.rootwrap<1.7.0,>=1.6.0 # Apache-2.0
 oslo.messaging<1.9.0,>=1.8.0 # Apache-2.0
 oslo.i18n<1.6.0,>=1.5.0 # Apache-2.0
 rfc3986>=0.2.0 # Apache-2.0
--- nova-2015.1.2/requirements.txt.orig	2015-10-13 07:52:44.000000000 -0700
+++ nova-2015.1.2/requirements.txt	2015-12-31 19:54:01.422821937 -0800
@@ -17,12 +17,10 @@ PasteDeploy>=1.5.0
 Paste
 sqlalchemy-migrate!=0.9.8,<0.10.0,>=0.9.5
 netaddr>=0.7.12
-paramiko>=1.13.0
 pyasn1
-Babel>=1.3
 iso8601>=0.1.9
 jsonschema<3.0.0,>=2.0.0
-python-cinderclient<1.2.0,>=1.1.0
+python-cinderclient<=1.3.1,>=1.1.0
 python-neutronclient<2.5.0,>=2.4.0
 python-glanceclient<0.18.0,>=0.15.0
 six>=1.9.0
@@ -35,7 +33,6 @@ oslo.log<1.1.0,>=1.0.0 # Apache-2.0
 oslo.serialization<1.5.0,>=1.4.0 # Apache-2.0
 oslo.utils!=1.4.1,<1.5.0,>=1.4.0 # Apache-2.0
 oslo.db<1.8.0,>=1.7.0 # Apache-2.0
-oslo.rootwrap<1.7.0,>=1.6.0 # Apache-2.0
 oslo.messaging<1.9.0,>=1.8.0 # Apache-2.0
 oslo.i18n<1.6.0,>=1.5.0 # Apache-2.0
 rfc3986>=0.2.0 # Apache-2.0
